test/promise-dialog_test.html

```diff
@@ -85,9 +85,10 @@
           });
           it('should resolve with value', function(done) {
             let expected = 42;
-            resolveButton.setAttribute('promise-value', expected);
+            resolveButton['promise-value'] = expected;
```

Comment:
I don't think this will account for binding. Isn't the value set in polymer as camelCase?